src/main/java/rs117/hd/utils/buffer/FixedLengthHashCode.java

```diff
@@ -1,10 +1,10 @@
-package rs117.hd;
+package rs117.hd.utils.buffer;
 
 public class FixedLengthHashCode {
```

Comment:
We can remove this class entirely now. I decided to move away from it and back to the standard library to keep hashing consistent throughout the codebase. Just forgot to delete it. :sweat_smile: